cloudigrade/api/clouds/aws/util.py

```diff
@@ -1051,6 +1052,14 @@ def update_aws_cloud_account(
         )
         cloud_account.delete()
 
+        with transaction.atomic():
+            user, created = User.objects.get_or_create(username=account_number)
```

Comment:
Just a general observation "thinking out loud" here. I think this is the the third instance in our code where we are `get_or_create`ing a user like this, and this is starting to feel like something we should wrap up in a helper function for more consistent reuse, testing, mocking, etc. I'm not suggesting you go change this now. Just planting the suggestion for maybe later. :)

Reply:
:+1: That sounds good, I'll merge this PR as is for now, because I've been waiting forever to close this story. But I'll add the refactoring to my todo list :)